src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1436,10 +1439,37 @@ await foreach (SourceFile source in context.store.Load(sessionId, loaded_files,
                     {
                         await OnSourceFileAdded(sessionId, source, context, token);
                     }
+
+                    if (AutoSetBreakpointOnEntryPoint)
+                    {
+                        var entryPoint = context.store.FindEntryPoint();
+                        if (entryPoint is not null)
+                        {
+                            // FIXME: id
+                            string bpId = "bp://99999";
```

Comment:
do we do anything specific with the bp id format?  If we're making one up can't we do something a little more obvious?